### DISCUSSION TITLE
Docs for contributing to Blitz codebase?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [blitz-js/blitz#2625] [General]
Hi all, are there any docs about the Blitz Monorepo and how everything works together because I would like to learn more about it and contribute, or is the package.json the only place I can look at and figure everything out myself? Thanks

**Comment by flybayer** [comment]
This page has all our documentation: https://blitzjs.com/docs/contributing There's also a video linked there that has an overview of the codebase.

But note that video is a bit dated because it does not include our Next.js fork. Our codebase is currently in flux because of the fork migration process (see https://github.com/blitz-js/blitz/pull/2516 for example)

Once this fork migration is complete, we'll make a new video, new docs, etc.

**Reply by ivanjeremic** [reply]
Any deadline or date on when the migration will be finished?

**Reply by flybayer** [reply]
Probably on the order of 1-2 months
